### ISSUE TITLE
Paste schema, to update or verify it (or verify JSON against a know schema)

### ISSUE THREAD

**Issue by bldesign** [jsonsystems/public#18]
Hi,

in the previous version (not sure which), it was possible to past the schema (paste JSON Schema on the right side). This possibility should remain as it provides several benefits:

1. Check a previously made JSON Schema against a JSON
1. Update a previously made JSON Schema 

Is that possible?

If not, I am forced to re-edit the infered JSON Schema even if I have it offline already perfectly done.

Anyway, as said, we do appreciate your contribution and it's a very attractive app.

**Comment by Radiergummi** [comment]
So... if I understand this correctly... jsonschema.net is currently an app to validate JSON against a single, arbitrary, read-only example schema, but every usefulness has been removed from the app piece by piece?

**Comment by jackwootton** [comment]
@Radiergummi Not quite. jsonschema.net is an application to generate JSON schema from JSON. It does not perform validation.
